setup.py

```diff
@@ -12,6 +13,17 @@
 
 exec(compile(version_file_contents, "pytools/version.py", "exec"), ver_dic)
 
+version = sys.version_info
+requirements = [
+          "decorator>=3.2.0",
+          "appdirs>=1.4.0",
+          "six>=1.8.0",
+          "numpy>=1.6.0",
+          ]
+
+if version[0] == 3 and version[1] == 6:
+    requirements.append("dataclasses>=0.7")
```

Comment:
I learned literally five minutes ago that [this](https://github.com/inducer/meshmode/pull/59#discussion_r504320314) is a possibility.

Reply:
Nice, that is much cleaner!